### ISSUE TITLE
Import sacctmgr dump into coldfront

### ISSUE THREAD

**Issue by buch01** [ubccr/coldfront#531]
I was unable to find information about importing a sacctmgr dumped config file into coldfront. I saw "slurm_check", which is able to compare the coldfront and sacctmgr configs and overwrite the sacctmgr config with coldfront's, but I am either using it incorrectly or it is unable to import the sacctmgr config. Is there a feature to import the sacctmgr config file into coldfront?

**Comment by dsajdak** [comment]
@buch01 No, that feature doesn't exist yet.  The slurm_check command is used for removing accounts and associations that are no longer part of an active allocation for a slurm related resource ([see here](https://coldfront.readthedocs.io/en/latest/howto/resources/slurm_setup/#removing-access-for-expired-or-revoked-allocations)).  There is nothing yet for importing an existing batch of slurm accounts/associations into ColdFront.
